src/components/textfieldButtons/js/TextfieldButtons.js

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2013 OCAD University
```

Comment:
Fix the copyright to 2016. This applies to all new or updated js files in this branch. For new files, use 2016, for updated ones, add the year 2016, for example, changing from 2013 to 2013-2016. Thanks.